src/index.css

```
@@ -651,18 +758,96 @@ section {
+}
+::-webkit-scrollbar-track {
+  background: var(--bg-secondary);
+}
+::-webkit-scrollbar-thumb {
+  background: var(--accent-gradient);
+  border-radius: 9999px;
+  border: 2px solid var(--bg-secondary);
+}
+
+/* Subtle animated aurora used behind hero / section dividers */
+@keyframes auroraShift {
```

Stylelint (17.12.0)

[error] Expected keyframe name "auroraShift" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)